coder-sdk/error.go

```diff
@@ -2,18 +2,20 @@ package coder
 
 import (
 	"encoding/json"
+	"errors"
 	"net/http"
 	"net/http/httputil"
 
 	"golang.org/x/xerrors"
 )
 
-// ErrNotFound describes an error case in which the requested resource could not be found
-var ErrNotFound = xerrors.Errorf("resource not found")
+// ErrNotFound describes an error case in which the requested resource could not be found.
+var ErrNotFound = errors.New("resource not found")
```

Comment:
I thought we were pro-xerrors?

Reply:
From chatting with @nhooyr we are using xerrors for the >go1.13 formatting/warps but also for the stack trace. Here we don't have any wrap/formatting, and as we are dealing with a "sentinel" error, having a trace indicated error.gol13 doesn't provide value and can lead to confusion.

Reply:
I think it's good to be consistent though. If we use `xerrors.New``, I think we're good.